pwiz_tools/Skyline/SettingsUI/Irt/CalibrateIrtDlg.cs

```diff
@@ -782,14 +822,21 @@ public void SetFixedPoints(int one, int two)
             comboMaxPeptide.SelectedIndex = two;
         }
         #endregion
+
+        private void gridViewCalibrate_CellFormatting(object sender, DataGridViewCellFormattingEventArgs e)
+        {
+            if (e.Value is double value && double.IsNaN(value))
+            {
+                e.Value = !double.IsNaN(value) ? value.ToString(Formats.RETENTION_TIME, CultureInfo.CurrentCulture) : string.Empty;

```

Comment:
Isn't it the case that double.IsNaN will always be false here?

Reply:
Good catch, fixed